<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTicket</class>
 <widget class="QWidget" name="AddTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backgroundGradient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>20</y>
     <width>1341</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x0:1, y2:1, x2:0, y2:1, stop:0 rgba(1, 15, 17, 255), stop:1 rgba(4, 15, 101, 255))</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="navBackground">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#010511;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="NavButton_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="profileButton">
        <property name="styleSheet">
         <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none;</string>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>5</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#32ACBE;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>15</number>
        </property>
        <property name="midLineWidth">
         <number>6</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="NavButton_9">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="assignButton">
        <property name="styleSheet">
         <string notr="true">font-size: 24px;font-family: Inter;color: #32ACBE; font-weight:bold; border:none;</string>
        </property>
        <property name="text">
         <string>Assigned</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_10">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>5</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#05448A;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>15</number>
        </property>
        <property name="midLineWidth">
         <number>6</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="NavButton_10">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="managementButton">
        <property name="styleSheet">
         <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none;</string>
        </property>
        <property name="text">
         <string>Management</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>5</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#32ACBE;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>15</number>
        </property>
        <property name="midLineWidth">
         <number>6</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="NavButton_11">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="styleSheet">
         <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none;</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_12">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>5</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#CA0736;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>15</number>
        </property>
        <property name="midLineWidth">
         <number>6</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="ticketTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>1131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;font-family: Inter;color: #040F65; font-weight:lighter; padding-right:5px; border:0px;padding-left:5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>New ticket title...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px; font-family: Inter;color: #D4F8F6; font-weight:lighter; border:none;</string>
   </property>
   <property name="text">
    <string>Created on: 1/1/1982 - </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="category">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>231</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;font-family: Inter;color: #040F65; font-weight:lighter; padding-right:5px; border:0px;padding-left:5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Category...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="extraData">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px; font-family: Inter;color: #D4F8F6; font-weight:lighter; border:none;</string>
   </property>
   <property name="text">
    <string>- Untitled MMO - Unresolved</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="details">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>1131</width>
     <height>441</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;font-family: Inter;color: #040F65; font-weight:lighter; border:5px solid white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="markdown">
    <string>**Description:**



**Steps to reproduce:**



**Thoughts:**


</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:24px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24px; font-weight:700;&quot;&gt;Description:&lt;/span&gt;&lt;span style=&quot; font-size:24px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:24px; font-weight:700;&quot;&gt;Steps to reproduce:&lt;/span&gt;&lt;span style=&quot; font-size:24px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:24px; font-weight:700;&quot;&gt;Thoughts:&lt;/span&gt;&lt;span style=&quot; font-size:24px;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="createTicket">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>720</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none; background-color:#32ACBE; border-radius:1px;</string>
   </property>
   <property name="text">
    <string>Create Ticket</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayedUser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>720</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none; background-color:transparent;font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; padding-left:5px; text-align:left;</string>
   </property>
   <property name="text">
    <string> Greg</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Image/Images/placeholderProfileIcon.png</normaloff>:/Image/Images/placeholderProfileIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 50px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none; background-color:transparent; text-align:left;</string>
   </property>
   <property name="text">
    <string>◄ Untitled MMO</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
